lib/graphql-error-handler.js

```diff
@@ -0,0 +1,17 @@
+let graphqlErrorHandler = null;
+
+if (process.env.NEW_RELIC_LICENSE_KEY) {
+  const newrelic = require('newrelic');
+  graphqlErrorHandler = error => {
+    newrelic.noticeError(error, { graphql: true });
```

Comment:
This looks good!

One thing to note: NewRelic says that you can tag error with the `noticeError(error, { tag: true })` API... but I couldn't find a way to filter it out in their UI. So instead I ended up having to create my own error "type". Which you can see [here](https://github.com/artsy/artsy-newrelic/blob/master/index.js#L13), but I think you can actually just set the `name` property of the error, e.g. `error.name = 'GraphQLError`, to achieve the same thing.